local/db/activity_program.sql

```diff
@@ -0,0 +1,39 @@
+drop table IF EXISTS activity_program;
+
+create table activity_programs (
+  id                 serial primary key,
+  user_id            varchar(255),
+  datetime           timestamp,
+  start_time         timestamp,
+  end_time           timestamp,
+  practice_section   varchar(255),
+  practice_Contents  varchar(255),
+  venue_id           serial,
+  route_id           serial,
+  contact_person_1   varchar(255),
+  contact_abstract_1 varchar(255),
+  contact_person_2   varchar(255),
+  contact_abstract_2 varchar(255),
+  create_time        timestamp,
+  update_time        timestamp,
```

Comment:
timestampは自動で設定できる様にお願いします。
また、PKの設定はまとめちゃっていいです。
```suggestion
  create_time        timestamp default CURRENT_TIMESTAMP,
  update_time        timestamp default CURRENT_TIMESTAMP,
  CONSTRAINT routes_PK PRIMARY KEY (id)
```